book/docs/Intro.ipynb

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
<p>I would not call out any specific proprietary software, though you could make a short list of common things. The important factors:</p><ul><li>license costs - would rather put that towards development and compute</li><li>scalable - coding is more scalable than a desktop or purely graphical solution</li><li class="ql-indent-1">as many as you want/need</li><li class="ql-indent-1">easily repeatable</li><li class="ql-indent-1">portable across systems</li></ul><p> </p><p>It should be more clear that scalable open source can be used in Academia, and is encouraged, but that the value can be even greater in Industry when the goal is to scale because of the ability to tap so much compute on-demand with limited licensing, and faster iteration cycles with community developed tools. Also even within your "company" you'll more likely be working with a team.</p><p> </p><p>It's not the norm in Industry to everything in the open either - really depends on the business model.</p><p> </p><p>Plenty, if not most NGOs also operate on grant type funding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/scaling_science/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated per this feedback @wildintellect thanks!

Reply:
<p>list didn't format nice, probably missing newline or similar</p>

Reply:
It for some reason in ReviewNB's rendering doesn't format nicely, but it does in GitHub natively. See the notebook in this branch:
<img width="764" alt="Screenshot 2024-04-09 at 1 37 41 PM" src="https://github.com/developmentseed/scaling_science/assets/18336080/890123b3-09ab-44e4-a4c0-5e6838f61575">